src/book/Chapter1-Agda.lagda.md

```diff
@@ -910,7 +910,7 @@ support `if..then..else` its prevalence is very subdued. In Agda, we have much
 better tools for branching, as we will soon see.
 
 This ascetic expression dedication precludes many other features that you might
-think be ubiquitous in programming languages. The most substantial of these
+think are ubiquitous in programming languages. The most substantial of these
```

Comment:
This one reads wrong to me in that I'd expect it to be the subjunctive. But maybe the best solution is to chop out the word entirely:

```suggestion
think ubiquitous in programming languages. The most substantial of these
```